File: Assignment1/sum1stDigi.java
import java.util.*;

public class sum1stDigi
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter how many numbers to enter: ");
		int n = sc.nextInt();
		int sum = 0;
		System.out.print("Enter " + n + " numbers: ");
		for (int i=0; i<n; i++)
		{
			int x = sc.nextInt();
			while (x>9)
			{
				x /= 10;
			}
			sum += x;
		}
		System.out.println(sum);

		sc.close();
	}
}

File: Assignment1/sum1stDigi.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Enter how many numbers to enter: ";
    cin >> n;
    int sum = 0;
    cout << "Enter " << n << " numbers: ";
    for (int i=0; i<n; i++) {
        int x;
        cin >> x;
        while (x>9) {
            x /= 10;
        }
        sum += x;
    }
    cout << sum;
    return 0;
}